code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -40,32 +68,99 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 \*||||||||||||||||||||||||||||||||||||||||*/
 
 /datum/gas
+	/// A reference to the cached gas metadata list
+	var/list/meta_gas_list
```

Comment:
What the fuck

Reply:
Easy reference for debugging exactly where you'd expect it to be without looking at the code. I might as well as long as I'm caching them.

Reply:
why cache them in the first place? I don't understand